packages/devtools_app/lib/src/inspector/inspector_tree_flutter.dart

```diff
@@ -536,14 +553,16 @@ class InspectorRowContent extends StatelessWidget {
       return const SizedBox();
     }
     Color backgroundColor;
-    if (row.isSelected || row.node == controller.hover) {
-      backgroundColor = row.isSelected
-          ? colorScheme.selectedRowBackgroundColor
-          : colorScheme.hoverColor;
+    if (row.isSelected) {
+      backgroundColor = errorText != null
```

Comment:
add a getter `hasError => errorText != null` and use that for all the errorText null checks